scripts/syslog-notify-oxidized.php

```diff
@@ -20,7 +20,7 @@
     oxidized_node_update($hostname, $msg, $matches['user']);
 } elseif (preg_match('/HWCM\/4\/CFGCHANGE/', $msg, $matches)) { //Huawei VRP devices CFGCHANGE syslog
     oxidized_node_update($hostname, $msg);
-} elseif (preg_match('/UI_COMMIT: User \\\\\'(?P<user>.+?)\\\\\' .*/', $msg, $matches)) {
+} elseif (preg_match('/UI_COMMIT_PROGRESS: Commit operation in progress: commit complete/', $msg, $matches)) {
     oxidized_node_update($hostname, $msg, $matches['user']);
```

Comment:
Hi

- The old message is still sent by other devices / versions ? if yes we should not remove it, but only add the COMMIT_PROGRESS one. 
- Is there a way to catch the user doing the change in the New message ? If no, then the oxidized_node_update should be updated accordingly